File: test_chat_api.py
import requests
import json

# 测试chat API
url = "http://localhost:8000/api/chat/query"
data = {
    "query": "Antinet",
    "conversation_history": []
}

try:
    response = requests.post(url, json=data)
    print(f"状态码: {response.status_code}")
    print(f"响应:")
    print(json.dumps(response.json(), ensure_ascii=False, indent=2))
except Exception as e:
    print(f"错误: {e}")

File: src/test_chat_api.js
// 模拟前端chatService的调用
async function testChatAPI() {
  const API_BASE_URL = 'http://localhost:8000/api/chat';
  
  const request = {
    query: '知识卡片',
    conversation_history: []
  };
  
  try {
    const response = await fetch(`${API_BASE_URL}/query`, {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
      },
      body: JSON.stringify(request)
    });
    
    const result = await response.json();
    console.log('Status:', response.status);
    console.log('Response:', JSON.stringify(result, null, 2));
    return result;
  } catch (error) {
    console.error('Error:', error);
    throw error;
  }
}

testChatAPI().then(console.log).catch(console.error);
